conans/client/cmd/export.py

```diff
@@ -260,11 +262,29 @@ def _capture_scm_auto_fields(conanfile, conanfile_dir, destination_folder, outpu
         output.success("Revision deduced by 'auto': %s" % scm_data.revision)
 
     local_src_path = scm.get_local_path_to_url(scm_data.url)
-    _replace_scm_data_in_conanfile(os.path.join(destination_folder, "conanfile.py"), scm_data)
+    _replace_scm_data_in_recipe(package_layout, scm_data, scm_to_conandata)
 
     return scm_data, local_src_path
 
 
+def _replace_scm_data_in_recipe(package_layout, scm_data, scm_to_conandata):
+    if scm_to_conandata:
+        conandata_path = os.path.join(package_layout.export(), DATA_YML)
+        conandata_yml = {}
+        if os.path.exists(conandata_path):
+            conandata_yml = yaml.safe_load(load(conandata_path))
+            if '.conan' in conandata_yml:
+                raise ConanException("Field '.conan' inside '{}' file is reserved to Conan usage.".format(DATA_YML))
```

Comment:
I this leading dot (".*") format any kind of convention for "protected" fields in the YAML format?

Reply:
No idea. Fields inside the YML are to be defined. I feel like storing any internal Conan fields under a `.conan` section might be useful, but this is just my proposal. I will add this comment to the PR description